py/selenium/webdriver/common/action_chains.py

```diff
@@ -370,6 +373,69 @@ def scroll(self, x: int, y: int, delta_x: int, delta_y: int, duration: int = 0,
         )
         return self
 
+    def set_active_pointer(self, pointer_input):
```

Comment:
This is very java-esque, can we just allow optional args to be passed in when instantiating?  Theres also alot of duplication in the logic here throughout all of them

Reply:
Do you mean that the logic of the 3 methods (`set_active_pointer, set_active_keyboard and set_active_wheel`) are the same and I should combine them in a single function with optional arguments?

I can do that but the three methods are kept for 3 different devices. Any suggestions for that?

Reply:
We shouldn't need to do "set active" at all. "Active" can be changed to whatever device is passed into the specific method.

Reply:
you can see what the Ruby bindings do for an example (though Ruby does other things differently like not creating default devices, etc).